helm/istio/charts/galley/templates/validatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,126 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
```

Comment:
This section contains no `ownerReferences` as the original `ValidatingWebhookConfiguration` did. Is the owner added in the code?

Reply:
I was assuming the operator would take care of that; but it explicitly doesn't for cluster-scoped resources: https://github.com/Maistra/istio-operator/blob/878bee2d374595b41bc5fa1ed6952a55f720b543/pkg/controller/servicemesh/controlplane/manifestprocessing.go#L129